README.md

```diff
@@ -2,6 +2,10 @@
 
 Hide users from user search on demand.
```

Comment:
The way this feature is described here ⬆ and how it was described in the sync doc below ⬇ was confusing,

> Implement a red list to hide specific users from the user directory
>
> *-- https://docs.google.com/document/d/1_EePsXw0UT87YTfh1IgMCJp-uku4twDkFGD-Y3OsRd4/edit#bookmark=id.r2bqwikkgih*

From some discussion in a comment thread there, it was explained as an end-user option instead of a way for someone else to control how people appear in the user directory.

> what we name "red list" is an option the end user may enable from the application settings to hide themself from the users directory. Other users will not see them during a search in this directory.
> 
> This name red list came from the French "liste rouge" which is defined for the home phone directory. This is not the same name in English (I forgot the actual translation)
> 
> *-- @giomfo, https://docs.google.com/document/d/1_EePsXw0UT87YTfh1IgMCJp-uku4twDkFGD-Y3OsRd4/edit?disco=AAAAWmvKV5U*


The new paragraph below explaining the details does clarify but it feels like this first part should also be updated. Perhaps:

```suggestion
Adds an option for end-users to hide themselves from the user directory.
```